tests/unit/FilingHistoryList2.spec.ts

```diff
@@ -60,7 +62,8 @@ filings.forEach((filing: any, index: number) => {
 
       // mount the component
       const $route = { query: {} }
-      wrapper = mount(FilingHistoryList, { store, mocks: { $route }, vuetify })
+      wrapper = mount(FilingHistoryList, {
+        store, mocks: { $route }, propsData: { dissolutionType: 'administrative' }, vuetify })
```

Comment:
This will make prop dissolutionType = 'administrative' for all tests in this for loop. You don't want that.

Instead, you can use `wrapper.setProps()` in your specific test blocks below.

Note that props exist for the life of the wrapper -- so other tests may see a changed prop -- so you may need to clear the prop after your test block.

Reply:
Good fix!